myapp/src/stores/TodoListStore.js

```diff
@@ -0,0 +1,63 @@
+import { observable, action, computed, configure } from "mobx";
+import todoListItem from "./../todoListItem.json";
+
+configure({ enforceActions: true })
+
+
+class TodoListStore {
+    @observable listItems = todoListItem;
+
+    @action
+    handleChangeCheckBox = (id) => {
+        const newTodoItems = this.listItems.map((element) => {
+            if (id === element.id) {
+                return {
+                    ...element,
+                    done: !element.done
+                }
+            }
+            return element;
+        });
+        this.listItems = newTodoItems;
+
+    }
+    //    handleChangeCheckBox = (index) => {
```

Comment:
if no need remove it.
For best practice don't let the other developers confused with your code comments, and let your comments only for a useful information.